com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -1230,8 +1233,11 @@ internal void SynchronizeNetworkObjects(ulong clientId)
                 ClientSynchEventData.OnWrite(nonNullContext.NetworkWriter);
 
                 var size = bufferSizeCapture.StopMeasureSegment();
+                var bytesReported = m_NetworkManager.LocalClientId == clientId
+                    ? 0
+                    : size;
```

Comment:
Late feedback on this part, but I wonder if it makes sense to add an internal utility function to network manager? Something like `m_NetworkManager.GetBytesForMetricReporting(clientId, size);`.

Reply:
I mean at that point it could be part of the BufferSizeCapture utility class

Reply:
Maybe, I don't know if BufferSizeCapture should know anything about client ids though. this eval only work if you know networkManager.LocalClientId

Reply:
This is one statement. I don't see too huge of a reason to refactor a single statement into a function

Reply:
I mainly proposed this because you're using the same pattern a bunch of times across the codebase, so it is a bit of a code smell. I agree that for such a small expression it's not a big deal